src/components/Pages/DetailsPage.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getDetailsFromAPI } from '../../redux/details';
+import '../../styles/details.css';
+
+const DetailsPage = () => {
+  const { state } = useParams();
+  const dispatch = useDispatch();
+
+  const data = useSelector((state) => state.details);
+  const item = data.slice(-1);
+  useEffect(() => {
+    dispatch(getDetailsFromAPI(state));
+  }, []);
+
```

Comment:
- [ ] It would be better if you add a button to go back to the home page. Please fix this minor issue. 😄 👍